src/core/Akka/Util/WildcardIndex.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Akka.Util
+{
+    public sealed class WildcardIndex<T> : IEquatable<WildcardIndex<T>> where T : class
```

Comment:
Shouldn't this be made `internal` too? Any reason why not?

Reply:
The original `WildcardTree<T>` was a public sealed class, I was just preserving that, though I don't see why it shouldn't be an internal class, since nothing uses it except for `Deployer`

Reply:
Changed to internal